### PR TITLE
feature: add Goodbye Page with styling

### PR DESCRIPTION
**Pull request by OlesiaTr**
- fixed spelling in the theme.js

### CODE REVIEW

**Comment by OlesiaTr** [comment]
This branch was created based on the feature/theming branch

**Comment by MaksVibe** [comment]
Resolve conflicts pls

**Comment by MaksVibe** [comment]
Some conflicts, fix it pls, so that I can make merge, thanks

File: components/GoodbyePage/GoodbyePage.tsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+"use client"
+import { Section, Title, Desc, Reminder } from "./GoodbyePage.styles"
+
+const GoodbyePage = () => (
+  <Section>
+    <Title>Дякуємо!</Title>
+    <Desc>Твоя анкета буде оброблена протягом доби</Desc>
+    <Reminder>Не забудь перевірити запрошення до Discord</Reminder>
```
Comment (MaksVibe) [inline components/GoodbyePage/GoodbyePage.tsx]: Screenshot 2023-06-29 at 14 47 01
Add an icon pls from design > approved

**Review by MaksVibe — CHANGES_REQUESTED** [review]

**Comment by OlesiaTr** [comment]
> Some conflicts, fix it pls, so that I can make merge, thanks

Resolved conflict

**Review by MaksVibe — APPROVED** [review]

**Pull request merged**